doc/RELEASE_PROCESS.rst

```diff
@@ -33,39 +33,72 @@ This is an outline of what needs to be done in order to release rst2pdf.
 
 #. Build manual
 
-   You will need the `White Rabbit`_ font installed. We check out the tag first so that the version
+   You will need the `White Rabbit`_ font installed. We check out the tag install via pip first so that the version
```

Comment:
I'm not sure this works. Is it "check out the tag AND install via pip" ? I think checking out the tag first links to the second half of the sentence which is why I'm confused.

Reply:
You're right. This is not good. I've taken another stab at it, including moving the font requirement to the top as that's where we mention the other dependencies required.